tests/test_extensions.py

```diff
@@ -139,6 +141,7 @@ def test_remote_v1rc4():
     ]
 
 
+@pytest.mark.skip("failing in CI but passing locally")
```

Comment:
What's the failure?

Reply:
I don't know exactly. Things were failing via the eo extension?

Reply:
It _looks_ like you've got it fixed, but LMK if you'd like me to take a look.

Reply:
Hi. I changed the name of the schema and it was giving aa JSONSchemaValidationError because of this. The error message wasn't helpful but it's working now.